### PR TITLE
Student can register interest for job

### LINKED ISSUE #5: Student can register interest for jobs
When I'm looking at new jobs posted on the site
So that I can let Will know I might want to apply
I want to register my interest for the jobs I like the look of

### PR DESCRIPTION
**Pull request by pitchinvasion**
Damn I hate javascript capybara

    Closes #5

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -53,12 +52,22 @@ group :development, :test do
   gem 'dotenv-rails'
   gem 'awesome_print'
   gem 'gist'
+  gem 'ffaker'
+  gem 'thin'
 end
 
 group :test do
+  gem 'connection_pool'
```
Comment (spike01) [inline Gemfile]: Is this used in the final code?

File: app/views/jobs/apply.rabl [hunk 2]
```diff
@@ -0,0 +1 @@
+{ text: "hello" }
```
Comment (spike01) [inline app/views/jobs/apply.rabl]: What are this?

File: config/environments/test.rb [hunk 3]
```diff
@@ -40,4 +40,6 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
   config.log_level = :fatal
+
+  #config.allow_concurrency = false
```
Comment (spike01) [inline config/environments/test.rb]: Needed?

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -12,55 +12,70 @@
...
+# get competing queries that can timeout and not get cleaned up
+#module MutexLockedQuerying
+#@@semaphore = Mutex.new
+
+#def async_exec(*)
+#@@semaphore.synchronize { super }
+#end
+#end
+
+#PG::Connection.prepend(MutexLockedQuerying)
```
Comment (spike01) [inline spec/rails_helper.rb]: Does this need to stay in?

File: spec/rails_helper.rb [hunk 5]
```diff
@@ -12,55 +12,70 @@
...
+  end
+
+  config.before(:each) do |example|
+    DatabaseCleaner.strategy = example.metadata[:js] ? :truncation : :transaction
+    DatabaseCleaner.start
+  end
+
+  config.after(:each) do
+    DatabaseCleaner.clean
+  end
```
Comment (spike01) [inline spec/rails_helper.rb]: Have we stopped transactional fixtures for good?

Comment (pitchinvasion) [inline spec/rails_helper.rb]: yes

**Pull request merged**